src/scripts/ic7/shared/htm_report.py

```diff
@@ -453,6 +453,40 @@ def get_combined_stats(self) -> dict[str, Any]:
             "Total Deaths averted (covid disruption) for hiv 2021 to 2026": total_deaths_averted,
         }
 
+    def comb_mort(self) -> pd.DataFrame:
+        """Generate graphs for the combined mortality. """
+
+        df = self._calculate_combined_mortality_stats()
+
+        return df['Combined mortality df']
+
+    def comb_inc(self) -> pd.DataFrame:
+        """Generate graphs for the combined incidence. """
+
+        df = self._calculate_combined_incidence_stats()
+
+        return df['Combined incidence df']
+
+    def comb_reduc(self) -> pd.DataFrame:
+        """Get reductions in mortality for IC and CF. """
+
+        df_mort = self._calculate_combined_mortality_stats()
+        del df_mort['Combined mortality df']
+        df_mort = pd.DataFrame.from_dict(df_mort, orient='index')
```

Comment:
this is fine. but just to note that you don't really need to use `del` because you could instead just reference the things from `df_mort` that you do want (rather than deleting the thing you don't and then using everything else). Very small point.

Reply:
now done